database/rrdset.c

```diff
@@ -1503,6 +1461,7 @@ void rrdset_done(RRDSET *st) {
         // and we have collected metrics for this chart in the past (st->counter != 0)
         // fill the gap (the chart has been just loaded from disk)
         if(unlikely(st->counter) && st->rrd_memory_mode != RRD_MEMORY_MODE_DBENGINE) {
+            // TODO this should be inside the storage engine
```

Comment:
Considering this comment, I will give a suggestion that I received in the past in another PR that I made. If this is not moved with this PR, please, create a issue for we never forget this.